test/unit/schedule/__snapshots__/SchedulerEditor.spec.js.snap

```diff
@@ -2754,8 +2753,7 @@ Object {
   "limit": Object {
     "endsAt": "",
```

Comment:
Should this be null?

Reply:
Not according to the base schedule object.
Will this cause an issue on the backend, sorry I am still playing catchup on the scheduler.

Reply:
Ok, it should be fine on the backend although null may be more consistent if that's what we're using elsewhere?

Reply:
@kingy04 Will this delay anything?

Reply:
The main thing is that the produced payload has this set to null if a date is not set, which the getSchedulerObject is doing, It wouldn't do any harm to have it set here too for consistency. It wont delay anything.

Reply:
Ok, I have set this now default to use null instead of an empty string.

Just as an aside and one to think about for the future. I think we need to be a bit more deliberate with our coding choices. Breaks in consistency should be deliberate.

Our code should serve two purposes, first to solve the problem and second to communicate to others in the team how we solved that problem. The use of an empty string for the inbound transformer vs the use of null in the outgoing transformer appears like a deliberate choice and that makes it harder for others to reason about changes like this.

Maybe something to bring up in the imminent 'What good looks like' talk.